### ISSUE TITLE
Unturned has some kind of a d3d11 issue.

### ISSUE THREAD

**Issue by TheBoxxd** [SmartlyDressedGames/Unturned-3.x-Community#1558]
I played Unturned a lot. But now,I just started up Unturned and it hides my cursor,shows up in the task bar and then just randomly closes.
I tried : Deleting and installing unturned,deleting all workshop mods and unsubscribing,trying to launch the game raw from the files and rebooting my pc.

https://drive.google.com/open?id=1-DZbcS-qzxmKnmvWfFhDMKUfMLAXhrbH

**Comment by SDGNelson** [comment]
Following up from #1546. Sorry I missed your comment!

Is that the full log file? It seems unusually short. You are right about the d3d11 warning: Are your graphics card drivers up-to-date?

Looking around a bit suggests the error may be related to running out of VRAM. It might also be worth trying to run the game with the `-force-glcore` command-line launch argument.

**Comment by TheBoxxd** [comment]
Yes,that is the full log file.

**Comment by TheBoxxd** [comment]
My cards are up to date and I'll try to run the game like that.

**Comment by TheBoxxd** [comment]
Hey,would you look at that! It launched! Tho,the fps is not as great as before,the game is now playable. (I tried going back to directx but it didn't work) Thanks Nelson! I'll leave this open if you can do anything else about the d3d11 issue!

**Comment by TheBoxxd** [comment]
Yeah,the FPS is really bad,any way of fixing the directx issue would be a god send

**Comment by SDGNelson** [comment]
Could you press `Win + R` to open the `run` dialog and then run `dxdiag` please? In the display tabs it should report how much VRAM your system has. (if that is the original issue described above)

Also: if your system has an integrated GPU and dedicated GPU you should double-check that the dedicated GPU is being used.

**Comment by TheBoxxd** [comment]
Dedicated GPU only,4096MB VRAM,the pc is good tho. Ran Unturned with everything on Ultra. Also my DirectX version us up-to date 12.

**Comment by TheBoxxd** [comment]
One more issue about OpenGL. I'm playing Unturned with Deferred graphics and that enables like,reflections and stuff. But OpenGL has to (I guess) make it's own which makes everything see through. So yeah. Still looking up to a fix.

**Comment by TheBoxxd** [comment]
I tried installing a DirectX 11 update. The game flashed a few times and started loading stuff. But then crashed (I still couldn't see anything)

**Comment by SDGNelson** [comment]
That does sound like a plenty powerful GPU. Even with direct X updated are you absolutely sure that your drivers are up-to-date? For example if you have an Nvidia GPU they have a "GeForce Experience" program which updates your drivers.

**Comment by TheBoxxd** [comment]
Yeah,I installed it and checked everything. Installed the Game Ready driver update and everything SHOULD work fine.

**Comment by TheBoxxd** [comment]
Hello..? Man...

**Comment by SDGNelson** [comment]
Sorry for the delay: I am not sure what else to recommend then. There is no helpful error information, and no similar errors to go off of. It seems your best option for the moment is to use OpenGL with the forward rendering setting. It might be patched in a future Unity LTS update.

**Comment by TheBoxxd** [comment]
Righto then. Thanks for the help anyways.

**Comment by TheBoxxd** [comment]
Hi,I'm back! After a lot of time (two weeks) I just thought of messing around with the files. I updated my directx with some old updates,screw around with unturned and etc. These did not work...Tho,I figured out that Unturned was very slow launching so maybe it was capped at some hardware usage. So I just figured...Let's add -high. Guess what,it worked. It flashed ton of times and got the game to work! I hope whoever is having this problem too should try putting -high in their launch!
